### DISCUSSION TITLE
Nơi thảo luận bài học tuần 3

### DISCUSSION THREAD

**Discussion by nguyenminhduc10a13PN** [nhom7PN/Nhom-7-C13#15] [General]
làm bài làm bài êyyyyy

**Comment by nguyenminhduc10a13PN** [comment]
today chúng ta sẽ phân công làm bài tin nhó các bạn @NguyenXuanPhong10a13pn @NguyenTanPhat10a13pn @NguyenTuanHuy10a13pn @NguyenTuanHuy10a13pn @Daoanhthu2711 #chayduacungmattroi

**Reply by ghost** [reply]
o`

**Comment by nguyenminhduc10a13PN** [comment]
@NguyenTanPhat10a13pn: Làm word về vấn đề mà thành viên trong nhóm đã chọn.
@Daoanhthu2711: Phụ trách mảng tìm nội dung cho bài nghị luận như là tìm câu tục ngữ, câu ca dao.
@NguyenTuanHuy10a13pn: Phụ trách mảng tìm nội dung cho bài nghị luận như là tìm câu tục ngữ, câu ca dao.
@NguyenXuanPhong10a13pn: Đảm nhiệm phần cuối: diễn giải những kĩ thuật được áp dụng ở đâu và thông số.

**Comment by NguyenTuanHuy10a13pn** [comment]
Ok đã rõ

**Comment by NguyenXuanPhong10a13pn** [comment]
oki ông ơi

**Comment by NguyenXuanPhong10a13pn** [comment]
@NguyenTanPhat10a13pn làm rồi gửi tui nào

**Reply by ghost** [reply]
tu72 tu72 nao`
